src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.css

```
@@ -0,0 +1,24 @@
+.permissionChoice {
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+  border-bottom: 1px solid rgba(54, 54, 54, 0.15);
```

There's a variable for this: `--grey-2`, please use that:
```suggestion
  border-bottom: 1px solid color-mod(var(--grey-2) alpha(15%));
```

---

Please use the variable `--grey-2` here as suggested in the previous review

```suggestion
  border-bottom: 1px solid color-mod(var(--grey-2) alpha(15%));
```